README.md

```diff
@@ -1,29 +1,112 @@
+                <table><tbody><tr><th>Name:</th><td>horreum url</td></tr><tr><th>Description:</th><td>Base URL for the Horreum server.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td><tr><th>Must match pattern:</th><td><code>((http|https)\:\/\/)?[a-zA-Z0-9\.\/\?\:@\-_=#]&#43;\.([a-zA-Z]){2,6}([a-zA-Z0-9\.\&amp;\/\?\:@\-_=#])*</code></td></tr></tbody></table>
+            </details><details><summary>horreum_username (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>horreum username</td></tr><tr><th>Description:</th><td>Username for the Horreum server.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+            </details><details><summary>test_access_rights (<code>enum[string]</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>test access rights</td></tr><tr><th>Description:</th><td>Access rights for the object being uploaded.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>enum[string]</code></td><tr><td colspan="2">
+        <details><summary>Values</summary>
+            <ul><li><strong><code>PRIVATE</code>:</strong> PRIVATE</li><li><strong><code>PROTECTED</code>:</strong> PROTECTED</li><li><strong><code>PUBLIC</code>:</strong> PUBLIC</li></ul>
+        </details>
+    </td>
+</tr></tbody></table>
+            </details><details><summary>test_name (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>test name</td></tr><tr><th>Description:</th><td>Name of the target test in Horreum for the object being uploaded.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+            </details><details><summary>test_owner (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>test owner</td></tr><tr><th>Description:</th><td>Owner of the object being uploaded.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+            </details><details><summary>test_start_time (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>test start time</td></tr><tr><th>Description:</th><td>Datetime formatted start time for the object being uploaded. Can also be provided as a JSONPath to a key in the document.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+            </details><details><summary>test_stop_time (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>test stop time</td></tr><tr><th>Description:</th><td>Datetime formatted stop time for the object being uploaded. Can also be provided as a JSONPath to a key in the document.</td></tr><tr><th>Required:</th><td>Yes</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
```

Comment:
I don't know what to do about this one. I think this can only be fixed in the docs generator. I can't really insert any formatting that will affect the HTML into the actual description in the schema.